src/components/app-header/app-header.tsx

```diff
@@ -9,6 +9,24 @@ export class AppHeader {
   @Prop() match: MatchResults;
   @Prop() linkUrl: string;
   @Prop() backgroundUrl: string;
+  @Prop() hideButton: boolean = false;
+
+  scrollElement;
+
+  componentDidLoad() {
+    if (this.hideButton) {
+      const button = document.getElementsByClassName(
+        'btn btn-primary btn-arrow'
+      );
+      button['0'].style.visibility = 'hidden';
+    }
```

Comment:
This will render the button, and "style" it to be hidden. Should prevent the button from rendering by applying this logic in the render function instead of the didLoad callback:

```
{this.hideButton
  ? null
  : <a class="btn btn-primary btn-arrow" onClick={this.scrollIntoView}>
      <i class="fa fa-arrow-down" aria-hidden="true" />
      <span class="sr-only">Scroll to content</span>
    </a>
}
```